marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubConfigImpl.java

```diff
@@ -1175,4 +1178,20 @@ private boolean getEnvPropBoolean(Properties environmentProperties, String key,
         }
         return res;
     }
+
+    @JsonIgnore
+    public String getInfo()
+    {
+        ObjectMapper objmapper = new ObjectMapper();
```

Comment:
It would be slightly better to pull objmapper out into the class as a private object so it doesn't have to be recreated every time the function is called.